### ISSUE TITLE
Export/import from file

### ISSUE THREAD

**Issue by PhoneixS** [maxiride/pf2e-encounters#54]
It could be nice to have the option to export and import encounters from a file.

"exporting" would be the same as downloading the file. And "importing" would be uploading a file. 

It could use a simple json format like one used with browser storage so both task could reuse part of the code.

This could be used as a more secure way to store encounters for the long term as browser's storage could be erased by accident more easily.

**Comment by maxiride** [comment]
Yes indeed, local storage or a downloadable file would simply be two different destinations.
ref for my future self: #52
